my-app/src/pages/home.tsx

```diff
@@ -0,0 +1,7 @@
+export default function Home() {
+  return (
+    <div className="d-flex justify-content-center h-100">
+      <h1 className="m-auto">Welcome!</h1>
```

Comment:
NIT: would like more than just welcome. Like, welcome to the game of thrones API thing. Etc. Doesn't matter at all as this is just a quick acedemic thing so.